### ISSUE TITLE
RIPAT

### ISSUE THREAD

**Issue by bioinfo16** [bioinfo16/Troubleshooting#3]
If you have any problem to run RIPAT ([https://github.com/bioinfo16/RIPAT](https://github.com/bioinfo16/RIPAT)), 
please contact to me directly or leave the error message or any relevances on this post.
__*(Strongly recommend attatching the comment to this issue.)*__

- Please check the version of RIPAT which you use. I recommend that users use the latest version.

**Comment by bioinfo16** [comment]
지금 사용하는 웹페이지가 특수 양식을 활용하고 있어서 사용했던 script를 그대로 복사해서 올리는 경우
특수문자를 명령어로 오인해서 왜곡되는 경우가 있습니다.
가능하면, script는 캡쳐화면으로 올려주시거나 script 파일을 올려주세요.
추가사항으로, '@'로 제 아이디 태그해서 올려주시면 저에게 메일로 전달됩니다.

**Comment by bioinfo16** [comment]
아래 링크 참고 부탁드립니다.
https://github.com/bioinfo16/Troubleshooting/issues/1#issuecomment-669604275

**Comment by Sminjeong96** [comment]
@bioinfo16

안녕하세요 김연주, 심지연학생과 함께 RIPAT test 진행하는 손민정입니다.
기존에 다운받은 패키지 모두 삭제하고 4.0 버전으로 다시 다운 받으려고 하는데 error가 떠서요..

![20200806_163941](https://user-images.githubusercontent.com/69286829/89504718-7228cf00-d803-11ea-8587-95d22af844f6.png)
![image](https://user-images.githubusercontent.com/69286829/89504840-a56b5e00-d803-11ea-9772-2bbb6152e22a.png)
![image](https://user-images.githubusercontent.com/69286829/89504964-d64b9300-d803-11ea-84c9-c1690629b2cb.png)

무엇이 문제인지 알려주시면 감사하겠습니다..

**Comment by bioinfo16** [comment]
@Sminjeong96 
안녕하세요. R은 패키지를 설치할 때 마다, 상관 없는 다른 패키지도 업데이트 할거냐고 묻습니다.
첫 capture 화면이 그건데, RIPAT과 상관없는 rlang이 업데이트가 들어가면서 이전에 있었던  rlang이 안지워지고 새로운게 안깔아진다는 의미로 에러가 발생한 겁니다.
RIPAT과 상관없으니 처음 화면과 같은 내용이 출력되면 숫자 입력하지 마시고 그냥 엔터치세요.

**Comment by bioinfo16** [comment]
@Sminjeong96 
혹시 데이터베이스 다운로드에 시간이 많이 걸리면, 제가 갖고 있는 데이터 파일 드리겠습니다.
데이터 파일 다운로드는 처음에 한번만 하면 되는거라
기존에 4PC에서 테스트 했기에 반복적으로 trial하지 않아도 될 것 같습니다.
아직 cpg나 repeat 다운로드 진행 중이시면 말씀해주세요.

**Comment by Sminjeong96** [comment]
@bioinfo16

RIPAT 다시 다운받고, 보내주신 데이터도 다운받은 후에  #4 진행중에 error가 발생했는데요.. 메세지창 이미지 첨부합니다.

![image](https://user-images.githubusercontent.com/69286829/89621251-f9417a00-d8cb-11ea-9070-cc2076f3ebd0.png)

![image](https://user-images.githubusercontent.com/69286829/89621288-0d857700-d8cc-11ea-87a8-0cc426fe32a8.png)

다운로드한 파일 위치를 넣었는데 혹시 파일경로가 잘못 된 것인가요.?

**Comment by bioinfo16** [comment]
@Sminjeong96 
네, 맞습니다. 제가 말씀드린 위치는 본래 데이터파일 다운로드 하던 위치 (extdata로 끝나는 경로)입니다.
makeData function 사용했을 때 폴더경로 떴을거예요.
해당위치로 파일 옮겨두고 진행하시면 됩니다.

**Comment by bioinfo16** [comment]
@Sminjeong96 
다시 진행하시기 전에 RIPAT 다시 한번 업데이트 부탁드립니다.
그 다음, makeData(organism = 'GRCh37', dataType = 'variant') 실행하신 후에 annotation 진행하시면 됩니다.
업데이트 방법은 아래와 같이 진행해주시면 되고, 기존에 드렸던 데이터 파일중에 GRCh37_clinvar.rds만 제외하고,
나머지 파일들은 RIPAT package extdata 폴더에 다시 옮겨주셔야 합니다.
https://github.com/bioinfo16/Troubleshooting/issues/1#issuecomment-668101908

**Comment by Sminjeong96** [comment]
안녕하세요 
김연주 학생과 함께 테스트 진행중입니다.
업데이트하고 방금 새로 보내주신 파일 load하고 makeDocument 실행하려는데  error가 나서 글 남깁니다.

![image](https://user-images.githubusercontent.com/69286829/92700883-4eb8ed00-f38a-11ea-97bc-d54fc3048251.png)

다운로드에 문제가 있는 것일까요?

**Comment by bioinfo16** [comment]
@Sminjeong96 
https://github.com/bioinfo16/Troubleshooting/issues/3#issuecomment-690076942
제가 나중에 다시 김연주학생에게 보낸 결과 파일로 실행하셨나요?
그러시면 res에 blast_gene_5K가 아니라 blast_gene5K로 들어가야 합니다.
콘솔에 ls()쳐보시면 리스트에 blast_gene5K로 나올거예요.

**Comment by Sminjeong96** [comment]
아 네네

![image](https://user-images.githubusercontent.com/69286829/92712475-ad836400-f394-11ea-803b-81e45b3ff882.png)


스크립트 수정하니까 파일 생성 잘 되는 것 같습니다!

**Comment by Sminjeong96** [comment]
![image](https://user-images.githubusercontent.com/69286829/92712530-cb50c900-f394-11ea-8f2d-becb7788eec7.png)


파일 이렇게가 전부가 맞을까요??

**Comment by bioinfo16** [comment]
@Sminjeong96 
네 맞습니다.

**Comment by Sminjeong96** [comment]
@bioinfo16 

https://github.com/bioinfo16/Troubleshooting/issues/2#issuecomment-690961192

심지연 tester와 같은 문제로 파일 다시 다운로드해서 진행하였는데 잘 돌아가는데 파일이 다르게 생성 되어서요

![image](https://user-images.githubusercontent.com/69286829/92897104-e3d5e780-f457-11ea-9486-cae7e54b7dfa.png)

심지연 tester와 같은 문제여서 동일하게 해결하였습니다.

![image](https://user-images.githubusercontent.com/69286829/92899239-98243d80-f459-11ea-9b38-be07419c79fa.png)
